src/cart/crt_hg.c

```diff
@@ -640,6 +984,27 @@ crt_hg_ctx_init(struct crt_hg_context *hg_ctx, int idx)
 
 	D_DEBUG(DB_NET, "crt_gdata.cg_share_na %d, crt_is_service() %d\n",
 			crt_gdata.cg_share_na, crt_is_service());
+
+	if (crt_gdata.cg_na_plugin == CRT_NA_OFI_PSM2) {
+		/*Create the hg_addr hash tables*/
+ 		rc = d_hash_table_create_inplace(D_HASH_FT_NOLOCK,
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs